modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.css

```diff
@@ -99,10 +107,14 @@
 		display: none;
 	}
 
-	.congrats-information .right-column .congrats-info-price {
+	.congrats-information .right-column .congrats-info-price  {
 		padding-top: 2rem;
 		text-align: left !important;
 	}
+	
+	.congrats-info-price-discount{
```

Comment:
Is there a father class for `.congrats-info-price-discount`. If there is, use it to encapsulate it.

Reply:
I realized that no element would be reused from the parent component, so I decided to work with a different class. I just changed the class name.

Reply:
Done!